stdlib/sys.mlp

```diff
@@ -17,10 +18,10 @@
    your changes will be lost.
 *)
 
-type backend_type = 
+type backend_type =
   | Native
   | Bytecode
-  | Other of string 
+  | Other of string
```

Comment:
Fabrice Le Fessant (2016/06/29 02:58 -0700):
> Yes, but removing whitespaces at end of line is always good, isn't it
> ?

Per se it is, definitely. One may prefere to do it in distinct commits,
though.